src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1198,6 +1389,145 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+"""
+Settings related to debugging why certain itineraries are returned.
+"""
+input PlanDebugSettingsInput {
+    """
+    Itinerary filter debug profile used to control the behaviour of itinerary filters.
+    """
+    itineraryFilterDebugProfile: ItineraryFilterDebugProfile = OFF
+}
+
+"""
+Plan location settings. Location must be set. Label is optional
+and used for naming the location.
+"""
+input PlanLabeledLocationInput {
+    """
+    A location that has to be used in an itinerary.
+    """
+    location: PlanLocationInput!
+
+    """
+    A label that can be attached to the location. This label is then returned with the location
+    in the itineraries.
+    """
+    label: String
+}
+
+"""
+Plan location. Either coordinates or a stop should be defined.
+"""
+input PlanLocationInput @oneOf {
+    """
+    Coordinates of the location. Note, either coordinates or a stop should be defined.
+    """
+    coordinates: PlanCoordinatesInput
+
+    """
+    Stop that should be used as a location for the search. The expected value is the stop's
+    feedscoped GTFS ID, for example `feedId:stopId`. Note, either coordinates or a stop should
+    be defined.
+    """
+    stop: FeedScopedId
+}
+
+"""
+Wrapper type for different types of preferences related to plan query.
+"""
+input PlanPreferencesInput {
+    """
+    Street routing preferences used for ingress, egress and transfers. These do not directly affect
+    the transit legs but can change how preferable walking or cycling, for example, is compared to
+    transit.
+    """
+    street: PlanStreetPreferencesInput
+
+    """
+    Transit routing preferences used for transit legs.
+    """
+    transit: TransitPreferencesInput
+
+    """
+    Accessibility preferences that affect both the street and transit routing.
+    """
+    accessibility: AccessibilityPreferencesInput
+}
+
+"""
+Street mode options for different phases of an itinerary.
+"""
+input PlanStreetModesInput {
```

Comment:
I thought about the `useDefaultTransit ` for a bit. I think there are a few potential issues:
I think it would be better to have the default transit on by default to ease the use of the API
  2. I'm not sure if it explicit enough about how you turn off transit